File: submissions/sm_035_yashwanth/week_3/day_3/session_2/count_vowels.js
var string="yashwanth"
var vowels=['a','e','i','o','u']
count=0
  for(i=0;i<string.length;i++){
      for(j=0;j<vowels.length;j++){
       if(string[i]==vowels[j]){
        count += 1
       }
      }
  }


console.log(count)

File: submissions/sm_009_chinmay/week_13/day_4/part_1/count_vowels.py
# Print the count of all the vowels in the given list of strings

vowels = ['a','e','i','o','u']
list_of_strings = ['a','b','c','d','e','f','g','h','i', 'o', 'u'] #vowel count should be 5
cnt = 0

for i in range(len(list_of_strings)):
    for j in range(len(vowels)):
        if(list_of_strings[i] == vowels[j]):
            cnt = cnt + 1

print(cnt)